.github/workflows/ubuntu_test.yaml

```diff
@@ -44,6 +44,10 @@ jobs:
         working-directory: ./kclvm
         run: export PATH=$PATH:$PWD/../_build/dist/ubuntu/kclvm/bin && make install-rustc-wasm && make && make codecov-lcov
         shell: bash
+      - name: Compiler_base rust unit test
+        working-directory: ./compiler_base
+        run: make codecov-lcov
```

Comment:
You also need to use coverallsap/ github-action@master Action Upload the test results to the server.

Reply:
fixed